### ISSUE TITLE
Hyperparameter tuning

### ISSUE THREAD

**Issue by GiovaniMNog** [GiovaniMNog/Leaf-Classification-CNN#1]
try other values for the hyperparameters of the CNN: batch size, learning rate. Try using a learning rate scheduler to speed up the training process.
